tests/nvapi_tests.cpp

```diff
@@ -14,8 +14,8 @@
 #include "../src/nvapi_mosaic.cpp"
 
 #include "nvapi_sysinfo_mocks.cpp"
-#include "nvapi_sysinfo_util.cpp"
```

Comment:
Is this change needed? Same with the change two lines below?

Reply:
I think it deserves a rename since it now handles Lfx as well which is not a sysinfo component. I can revert and keep the filename if desired.

Reply:
May be I'm missing it, but as far as I'm seeing it, there are no renames in the test files. The other renames should be covered with the existing includes in this file.

Reply:
4Ah, scratch that. I'm seeing the rename now, missed it because I had flagged the file as already seen. All good!